docs/rules/no-confusing-set-timeout.md

```diff
@@ -0,0 +1,61 @@
+# Disallow using confusing setTimeout in test (`no-confusing-set-timeout`)
+
+<!-- end auto-generated rule header -->
+
+This rule will raise a warning about confusing `jest-setTimeout` in test.
+
+## Rule details
+
+This rule describes some tricky ways about `jest.setTimeout` that should not
+recommend in Jest:
+
+- should set `jest.setTimeout` in any testsuite methods before(such as
+  `describe`, `test` or `it`);
+- should set `jest.setTimeout` in global scope.
+- should only call `jest.setTimeout` once in a single test file;
+
+Examples of **incorrect** code for this rule:
+
+```js
+describe('A', () => {
```

Comment:
Can you expand these examples so they're not just using single letters for the titles